src/supermarket/app/views/cookbooks/_sidebar.html.erb

```diff
@@ -90,6 +90,16 @@
     <h4><i class="fa fa-key"></i> License</h4>
     <p><%= version.license %></p>
 
+    <% if version.chef_versions.present? %>
+      <h4><i class="fa fa-key"></i> Chef Versions</h4>
```

Comment:
We can find something more appropriate later, maybe after upgrading the version of font-awesome we use.